[Nanoleaf] ipv6 address doc (#15612)
openhab/openhab-addons: bundles/org.openhab.binding.nanoleaf/README.md

| refreshInterval | Interval in seconds to refresh the state of the light panels settings. Default is 60. |
| deviceType      | Defines the type `lightpanels` (triangle) or `canvas` (square or hexagon)             |

> **Important note on the topic of IPV6 ip addresses:**

Reviewer: ```suggestion
**Important note on the topic of IPv6 addresses:**
```

> Since at least firmware 8.5.2 panels may start to go offline and online. 

Reviewer: ```suggestion
With firmware version 8.5.2 or newer, panels may change between being OFFLINE and ONLINE. 
```

> After some investigation it turned down that the devices are discovered as IPV6 devices which then cannot be used correctly within the binding to run the API requests to the devices.

Reviewer: ```suggestion
This is due to the fact that if they are discovered with IPv6 addresses, the binding is not able to correctly send API requests to the devices.
```

> For the time being it is therefore recommended, if this is an option for you, to disable IPV6 on the openhab server. 

Reviewer: ```suggestion
It is therefore recommended to disable IPv6 on the openHAB server. 
```

> For example this can be achieved on openhabian with the following

Reviewer: ```suggestion
This can e.g. be achieved on openHABian the following way:
```

```text
sudo nano /etc/sysctl.conf`

Add the following at the bottom of the file:

net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1

```

> then reboot.

Reviewer: ```suggestion
Reboot your server after the change.
```
